modules/ui/panels/history.js

```diff
@@ -200,7 +200,7 @@ export function uiPanelHistory(context) {
             .attr('href', 'https://pewu.github.io/osm-history/#/' + entity.type + '/' + entity.osmId())
             .attr('target', '_blank')
             .attr('tabindex', -1)
-            .html('PeWu');
+            .html('History');
```

Comment:
The link text is no longer the name of the tool itself, so now it’s more important to localize it into the user’s language. To make the link text localizable, call the `t()` method, passing in a string ID like `info_panels.history.pewu_link_text`. Then add an entry around here called `pewu_link_text` and set it to the string `History`:

https://github.com/openstreetmap/iD/blob/3e7fd6e0330a14319206c9c9eb47afb3d97ca8d9/data/core.yaml#L653

Reply:
Can we rethink this? 
I actually do think it's better to have these link texts to be the name of the tool itself.  
These information panels are expected to be used by more skilled users.

Reply:
Makes sense 👍🏻. I'll close this PR